lib/rules/base.js

```diff
@@ -69,6 +73,18 @@ module.exports = class Base {
     return this._configResolver.editorConfig();
   }
 
+  get _moduleName() {
+    if (!loggedModules.has(this.ruleName)) {
+      this._console.log(
+        'The `_moduleName` property is deprecated. Please use `_filePath` instead.'
+      );
+
+      this.loggedModules.push(this.ruleName);
```

Comment:
This doesn't seem right, it is a module scoped variable (not an instance property)

Reply:
Derp. Mistake.